logz/logz.go

```diff
@@ -9,16 +9,28 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func Gvk(gvk schema.GroupVersionKind) zapcore.Field {
```

Comment:
Group kinds for both controller and controlled object are mentioned in the logs using fixed field names. Same with object names.
No need to log versions, only group and kind are needed.